onnxruntime/core/framework/utils.cc

```diff
@@ -647,7 +645,7 @@ void DumpNodeOutputs(OpKernelContext& context, const Node& node, const SessionSt
                 }
               }
 #else
-  ORT_UNUSED_PARAMETER(session_state);
+              ORT_UNUSED_PARAMETER(session_state);
```

Comment:
nit: seems like some formatting issue ? (This line itself doesn't seem to have any change)

Reply:
This is how clang-format automatically formats the file, so at some point it was updated without being formatted correctly.